lambdas/data-migration1/tests/test-collections.js

```diff
@@ -49,7 +43,14 @@ const generateFakeCollection = (params) => ({
 let collectionsModel;
 let rulesModel;
 
-test.before(async () => {
+test.before(async (t) => {
+  t.context.knexAdmin = await getKnexClient({
+    env: {
+      ...localStackConnectionEnv,
+      migrationDir: `${path.join(__dirname, '..', '..', 'db-migration', 'dist', 'lambda', 'migrations')}`,
```

Comment:
🐜 Since you aren't doing migrations with this connection, I don't think you need to specify `migrationDir` here.

Reply:
But we are doing migrations in this test? That's the whole reason for this setup

https://github.com/nasa/cumulus/pull/1843/files#diff-30596c5988ea3c50cac49ff87ca9edbeR77

Reply:
That's the admin connection.     The migration is happening as a user connection, from the knex at L69